code/modules/client/preference/preferences.dm

```diff
@@ -2050,7 +2047,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 
 				if("ghost_radio")
 					toggles ^= CHAT_GHOSTRADIO
-				
+
```

Comment:
WHAT THE HELL ARE YOU

Reply:
Probably your IDE cleaning up extra spaces

Reply:
I use DM

Reply:
Can I recommend VSCode? It has:
- Global search across workspace
- Allows you to jump to proc and bar definitions even if in separate files
- Has built in git so you don’t need another client
- Dark Mode

That’s just the surface

*100% not a paid representative*